tensorflow_estimator/python/estimator/estimator.py

```diff
@@ -1076,8 +1076,14 @@ def _call_input_fn(self, input_fn, mode):
       kwargs['params'] = self.params
     if 'config' in input_fn_args:
       kwargs['config'] = self.config
-    with ops.device('/cpu:0'):
+    if input_context and 'input_context' in input_fn_args:
+      logging.info('`input_context` has been detected in `input_fn`.'
```

Comment:
Maybe logging 'the `input_fn` accepts an `input_context` which will be given by Distribution Strategy.'?

Reply:
Things have been changed.